atlp-rwanda/tech-strikers-bn-backend
# #175636829 Configure BareFoot Nomad to use PostgreSQL as DB Engine

**What does this PR do?**

Connect BareFoot Nomad to use PostgreSQL as DB Engine

**Description of Task to be completed?**
Have the following credentials for accessing database:
  user: "andela",
  host: "localhost",
  database: "barefootnomad_db",
  password: "andela2020",
  port: "5432"

**How should this be manually tested?**
After cloning the repo, cd into it,  run `git checkout ft-postgressConfig` then run _`npm start_`
.use postman to insert the first name and check in the database if the user is there.
use post method and put the following schema in the body:
```
{
    "first_name": "Thierry"
}
```

After running postgress on your local machine, connect to barefootnomad_db and run this comand:
`SELECT * FROM visitors;`

**Any background context you want to provide?**
Add one post route to ensure the verification if the app is connected to the database. The route is still in the app.js. It is not written the user-route file. 

**What are the relevant pivotal tracker stories?**
[#175636829](https://www.pivotaltracker.com/n/projects/2475704/stories/175636829)
#### Screenshots (if appropriate)

<img width="1145" alt="on postman" src="https://user-images.githubusercontent.com/56275083/98851228-a34a1880-245e-11eb-90cd-f11bac2951a2.png">
<img width="357" alt="In database" src="https://user-images.githubusercontent.com/56275083/98851346-cc6aa900-245e-11eb-95b0-b17e88cc1166.png">
